modules/gis/src/lib/geojson-to-binary.ts

```diff
@@ -129,11 +132,9 @@ function firstPass(features: Feature[]): FirstPassData {
 
         // If property has not been seen before, or if property has been numeric
         // in all previous features, check if numeric in this feature
-        // If not numeric, false is stored to prevent rechecking in the future
-        numericPropKeys[key] =
-          numericPropKeys[key] || numericPropKeys[key] === undefined
-            ? isNumeric(val)
-            : numericPropKeys[key];
+        // If not numeric, Array is stored to prevent rechecking in the future
+        // Additionally, detects if 64 bit precision is required
+        propArrayConstructors[key] = deduceArrayType(val, propArrayConstructors[key]);
```

Comment:
Nit: Should we be consistent with type vs constructor? I personally find the "constructor" naming confusing though technically it is arguably more "correct".

```suggestion
        propArrayTypes[key] = deduceArrayType(val, propArrayConstructors[key]);
```
```suggestion
        propArrayConstructors[key] = deduceArrayConstructor(val, propArrayConstructors[key]);
```

Reply:
I don't like the "Constructor" naming much either, I initially put it there as it was consistent with what Typescript had for the types, but in my mind it is better to talk about "types" as that is what code is doing. I will standardize on "Type"